src/pymorize/pipeline.py

```diff
@@ -246,6 +246,7 @@ class DefaultPipeline(FrozenPipeline):
         "pymorize.timeaverage.compute_average",
         "pymorize.units.handle_unit_conversion",
         "pymorize.caching.manual_checkpoint",
+        #"pymorize.global_attributes.set_global_attributes",
```

Comment:
Why is this step commented out? Also, this should happen before the checkpoint, not after

Reply:
Yeap, agreed.

Reply:
working on this